### DISCUSSION TITLE
Zeitverzögerung bei arm64

### DISCUSSION THREAD

**Discussion by m1k3f15h** [mrbungle64/ioBroker.ecovacs-deebot#375] [Q&A]
Hallo Leute 😊

Ich habe meinen IoBroker von einem Pi4 mit arm auf einen Pi4 mit arm64 umgezogen (Docker)

Mir ist nun aufgefallen, dass die Befehle an den Roboter auf der arm64 Version um etliches länger dauern, bis sie ausgeführt werden (teilweise bis zu mehrefen Minuten)

Teste ich den selben befehl auf dem Pi4 mit 32bit System geht es in Sekundenschnelle.

Kann das jemand bestätigen?

Mfg
Mike

**Comment by mrbungle64** [comment]
Hallo @m1k3f15h 

meine 4 Adapter-Instanzen laufen auf problemlos mit arm64 (Raspberry Pi 3b, 64-Bit Raspberry Pi OS).

Allerdings nicht in einem Docker Container.
Ich tippe eher drauf, dass es was mit der Docker Installation zu tun hat. Ist aber reine Spekulation 😉

**Reply by m1k3f15h** [reply]
Servus @mrbungle64,

Danke schonmal für deine Rückmeldung.
Vielleicht meldet sich ja noch jemand hier, der das selbe Problem hat oder bestätigen kann, dass es bei ihm auch in Docker funktioniert.

mfg
Mike
